src/AbstractPolyhedron_functions.jl

```diff
@@ -211,3 +212,126 @@ function remove_redundant_constraints(constraints::AbstractVector{LinearConstrai
         return EmptySet{N}()
     end
 end
+
+"""
+    linear_map(M::AbstractMatrix{N},
+               P::AbstractPolyhedron{N};
+               check_invertibility::Bool=true,
+               cond_tol::Number=DEFAULT_COND_TOL,
+               use_inv::Bool=!issparse(M)
+               ) where {N<:Real}
+
+Concrete linear map of a polyhedron in constraint representation.
+
+### Input
+
+- `M` -- matrix
+- `P` -- polyhedron in constraint representation
+- `check_invertibility` -- (optional, deault: `true`) check if the linear map is
+                           invertible, in which case this function uses the matrix
+                           inverse; if the invertibility check fails, or if
+                           this flag is set to `false`, use the vertex representation
+                           to compute the linear map (see below for details)
+- `cond_tol` -- (optional) tolerance of matrix condition (used to check whether
+                the matrix is invertible)
+- `use_inv`  -- (optional, default: `false` if `M` is sparse and `true`
+                otherwise) whether to compute the full left division through
+                `inv(M)`, or to use the left division for each vector; see below 
+
+### Output
+
+The type of the result is "as close as possible" to the the type of `P`.
+To fix the type of the output to something different than the default value,
+consider post-processing the result of this function with a call to a suitable
+`convert` method.
+
+In particular, the output depends on the type of `P` and the algorithm that was used:
+
+- If the vertex-based approach was used:
+    
+    - If `P` is a `VPolygon` then the output is a `VPolygon`.
+    - If `P` is a `VPolytope` then the output is a `VPolytope`.
+    - Otherwise, the output is a `VPolygon` if the dimension of `P` is `2`, or a
```

Comment:
```suggestion
    - Otherwise, the output is a `VPolygon` if the dimension of `P` is two and a
```
(to make it consistent with the other paragraph)